pages/blog/2020-03-12-nextjs-without-client-side-react.md

```diff
@@ -0,0 +1,164 @@
+# Server Side Only React
+
+---
+
+<Intro>
+A commonly heard critique of React is that it's an over-engineered abstraction, that for the vast majority of websites provide little value. I tend to agree, so I removed it... on the client.
+</Intro>
+
+I see a lot of websites that have little or no interactivity on the client, yet somehow justifies sending me 2 MB of JavaScript, which makes my mobile device crawl and behave like it's taking its last dying breath.
+
+Regardless, when I re-built my personal website (for the n:th time) I made the decision to use [React](https://reactjs.org/) and [Next](https://nextjs.org/)! Am I a hypocrite? (You decide for yourself).
+
+<Disclaimer>Disclaimer: This article should not be considered as general advice or best practice in any way. This is an experiment of a mad man. OK? Carry on.</Disclaimer>
+
+[![Serverless Server side Rendering](/hello-world-react.jpg "This is me")](https://twitter.com/thomasfuchs/status/708675139253174273)
+
+## Why Next + React for a Blog?
+
+- I wanted to write blog posts in Markdown, so I needed something to convert Markdown to HTML. [MDX](https://mdxjs.com/) (Markdown + JSX) enabled me to transform Markdown into custom React Components.
+
+- React aside, I really like the component mental model and composition. I wasn't keen on going back to string based templates and CSS pre-processors (warning: looking back at my gulp/grunt files can trigger nightmares).
```

Comment:
"I really like the component mental model and composition." Feels aborted when you just say composition. Also, what composition are refering to. This could be written like this:

I really like the component and composition mental model.
I really like the component mental model and their composition.